.github/workflows/build.yml

```diff
@@ -7,23 +7,19 @@ jobs:
 
     strategy:
       matrix:
-        compiler: [gcc-7, gcc-8, gcc-9, gcc-10, clang-9, clang-10]
+        compiler: [gcc-9, gcc-10, gcc-11, clang-12, clang-13, clang-14]
         bits: [32, 64]
```

Comment:
Perhaps it would be better for consistency with LTO to use `bits: [M32=y, M32=n]`, and change scan-build line accordingly.

Reply:
I've addressed that differently in a followon PR, to work around an issue with CodeQL